### PR TITLE
Updated Chinese(simple) translation for v1.6.1

### PR DESCRIPTION
**Pull request by shaolin-kongfu**
### Check if this PR fulfills these requirements:
* [x] My changes respect the [Kodi add-on development rules](https://kodi.wiki/view/Add-on_rules)
* [x] I have read the [**CONTRIBUTING**](Contributing.md) document.
* [x] I made sure there wasn't another one [Pull Requests](../../pulls) opened for the same update/change
* [x] I have successfully tested my changes locally

#### Types of changes
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Feature change (non-breaking change which change behaviour of an existing functionality)
- [x] Improvement (non-breaking change which improve functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: resources/language/resource.language.zh_cn/strings.po [hunk 1]
```diff
@@ -0,0 +1,1027 @@
...
+
+msgctxt "#30031"
+msgid "General"
+msgstr "常规"
+
+# Unused 30032
+
+msgctxt "#30033"
+msgid "Enable Dolby Digital Plus (DDPlus HQ and Atmos on Premium)"
+msgstr "启用杜比 Plus"
```
Comment (CastagnaIT) [inline resources/language/resource.language.zh_cn/strings.po:145]: Here is needed to specify also: DDPlus HQ and Atmos on Premium

**Comment by CastagnaIT** [comment]
everything else is fine,
thanks for your contribution

File: resources/language/resource.language.zh_cn/strings.po (hunk 1 above)
Comment (shaolin-kongfu) [reply resources/language/resource.language.zh_cn/strings.po]: OK,I have fixed it.

Comment (CastagnaIT) [reply resources/language/resource.language.zh_cn/strings.po]: to be safe i just want to try explain better
DDPlus HQ and Atmos sound type are both enabled only with "Premium netflix account"
is written only "premium" to be concise

Comment (shaolin-kongfu) [reply resources/language/resource.language.zh_cn/strings.po]: I just get your point now.
How about translate this sentence like this:"Enable DDPlus HQ and Atmos for Premium accounts"
Is it ok?

**Comment by CastagnaIT** [comment]
Hi,
for the new add-on release 1.7.0, there are some changes to do

to make things easier for you i have prepared your file updated, so you will have only to add the missing translations
[strings.po.txt](https://github.com/CastagnaIT/plugin.video.netflix/files/4975244/strings.po.txt)

Then rebase your branch of the PR and add the po changed
(you can squash all previous commits to one or delete them)
thanks

**Pull request closed without merging**